src/modules/workspace/hooks/useWorkspace.tsx

```diff
@@ -4,63 +4,73 @@ import { useCallback, useState } from 'react';
 import { MdOutlineError } from 'react-icons/md';
 import { useNavigate, useParams } from 'react-router-dom';
 
-import { useAuth } from '@/modules/auth/hooks/useAuth';
 import { useHomeDataRequest } from '@/modules/home/hooks/useHomeDataRequest';
 import { useNotification } from '@/modules/notification';
 import { useTransactionsSignaturePending } from '@/modules/transactions/hooks/list';
 
 import { Pages } from '../../core';
 import { PermissionRoles } from '../../core/models';
-import { useGetCurrentWorkspace } from '../hooks/useGetWorkspaceRequest';
+import { useGetWorkspaceRequest } from '../hooks/useGetWorkspaceRequest';
 import { useSelectWorkspace } from './select';
 import { useGetWorkspaceBalanceRequest } from './useGetWorkspaceBalanceRequest';
 import { useUserWorkspacesRequest } from './useUserWorkspacesRequest';
+import { IUseAuthReturn } from '@/modules/auth/services';
 
 const VAULTS_PER_PAGE = 8;
 
 export type UseWorkspaceReturn = ReturnType<typeof useWorkspace>;
 
-const useWorkspace = () => {
+const useWorkspace = (authDetails: IUseAuthReturn) => {
   const navigate = useNavigate();
   const { workspaceId, vaultId } = useParams();
 
-  const auth = useAuth();
-
   const [visibleBalance, setVisibleBalance] = useState(false);
 
   const toast = useNotification();
   const workspaceDialog = useDisclosure();
+  const pendingSignerTransactions = useTransactionsSignaturePending();
 
-  const workspaceHomeRequest = useHomeDataRequest();
+  const worksapceBalance = useGetWorkspaceBalanceRequest(
+    authDetails.userInfos?.workspace?.id,
+  );
+
+  const predicatesHomeRequest = useHomeDataRequest(
+    authDetails.userInfos?.workspace?.id,
+  );
   const userWorkspacesRequest = useUserWorkspacesRequest();
```

Comment:
we dont need of this info here, call this hook just on header component

Reply:
Done